documentation/pe_puppetdb_notes.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: "PuppetDB 3.2 » PE PuppetDB Notes"
+layout: default
+canonical: "/pe-puppetdb-notes"
+---
+
+[reports]: ./api/query/v4/reports.html
+[terminus]: ./puppetdb_connection.markdown#include_unchanged_resources
+
+## 2015.3.0
```

Comment:
Hmm, there are other large-set features that might span multiple config areas that aren't PE btw. Like structured facts, trusted nodes. Just saying, these concepts of 'change these 3 things to turn on this featureset' aligns better with my thinking, and having a PE only section isn't bad either. I dunno I dunno, I need to ponder this some more. I think I know what you're trying to achieve though.